java/src/main/java/io/vertx/ext/eventbus/client/logging/Logger.java

```diff
@@ -0,0 +1,29 @@
+package io.vertx.ext.eventbus.client.logging;
+
+public interface Logger {
```

Comment:
logging in general should not be public and should go in an impl package

Reply:
I created an impl package. Yet, not all classes could be made package private as `JULLoggerFactory` within `impl` is being referenced from `LoggerFactory` (parent ns) and `LoggerFactoryImplBase` needs to be accessed by the other namespace (wherever it is located), too.

Reply:
If all logging classes would reside in the same package (no `impl` package), I could make those two private.